app/models/usergroup.rb

```diff
@@ -36,25 +36,25 @@ class Usergroup < ActiveRecord::Base
   # This methods retrieves all user addresses in a usergroup
   # Returns: Array of strings representing the user's email addresses
   def recipients
-    all_users.map(&:mail).flatten.sort.uniq
+    all_users.map(&:mail).flatten.uniq.sort
```

Comment:
since this is in memory sort/uniq, is it actually better performance? I always assumed in order to uniq it effectively you need to sort it first anyway? a quick google did not reveal that much (http://stackoverflow.com/questions/6712046/ruby-sort-and-uniq-which-to-run-first)